src/main/scala/io/pinecone/spark/pinecone/PineconeDataWriter.scala

```diff
@@ -1,7 +1,7 @@
 package io.pinecone.spark.pinecone
 
 import io.pinecone.proto.{UpsertRequest, Vector => PineconeVector}
-import io.pinecone.{PineconeClient, PineconeClientConfig, PineconeConnection, PineconeConnectionConfig, PineconeException}
+import io.pinecone.{PineconeClient, PineconeClientConfig, PineconeConnection, PineconeConnectionConfig}
```

Comment:
Just to confirm, removed because it's not needed in this file?

Reply:
Correct!